packages/storybook8/stories/Composites/CallWithChatComposite/Docs.mdx

```diff
@@ -0,0 +1,197 @@
+components in this UI library.
+
+## Custom Branding
+
+Along with applying a Fluent Theme to style the composites, you can also inject your own custom branding. You
+can inject a background and logo into the Composite configuration page to present to your users. This is done by
+passing background and logo properties to the `options` of the Composite.
+
+**Image recommendations.** The background image should be as simple and uncluttered as possible to avoid text
+becoming unreadable against the background. The background image should have a minimum size of 576x576 pixels
+and a maximum size of 2048x2048 pixels. The recommended size is 1280x720 pixels. The recommended size for the
+logo image is 128x128 pixels.
+
+<Stack horizontalAlign="center">
+  <img
+    style={{ width: '100%', maxWidth: '50rem' }}
+    src="images/composite-logo-background.png"
+    alt="CallWithChatComposite with a logo and background applied"
+  />
+</Stack>
+<Source
+  code={`
+<CallWithChatComposite options={{
+  branding: {
+    logo: {
+      url: 'https://...',
+      alt: 'My company logo',
+      shape: 'circle'
+    },
+    backgroundImage: {
+      url: 'https://...'
+    }
+  }
+}} />
+`}
+/>
+
+## Customize Call Controls
+
+CallWithChatComposite provides a set of default controls for the call that can be customized similar to
+CallComposite. Check out [Customize Call Controls](./?path=/docs/composites-callcomposite--docs#customize-call-controls)
+to learn more.
+
+## Adding image sharing
+
+### Inline Image In Teams Interop Meeting Chat Thread
+
+<SingleLineBetaBanner />
+The Azure Communication Services end user can receive images shared by the Teams user without any additional setup when
+using the `CallWithChat` Composite in a Teams interop scenario,. Azure Communication Services end user would need to
+join the Teams meeting first, as soon as the Teams user sends a file from the Teams client, the Azure Communication
+Services end user will be see the image in the chat thread. Please check out the tutorial for [Enable inline image using
+UI Library in Teams Interoperability
+Chat](https://learn.microsoft.com/en-us/azure/communication-services/tutorials/inline-image-tutorial-interop-chat) for
+details.
+
+## Adding file sharing
+
+### File Sharing In Azure Communication Services Chat Thread
+
+<SingleLineBetaBanner />
+
+In a chat thread where all participants are Azure Communication Services end users, the `CallWithChat`
+Composite supports file capabilities in conjunction with your choice of a storage solution. Using the provided
+APIs, you can composite to support uploading files and displaying them on the send box before sending, and the
+once they have been sent or received. For an end to end tutorial on enabling file sharing with Azure Blob
+Storage as your choice of a storage solution, refer to our
+[tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial).
+
+<Source
+  code={`<CallWithChatComposite
```

Comment:
should this be moved to snippet, since this is a long snippet?

Reply:
It doesn't compile by itself so it would fail CI if it was a separate snippet?